features/trainee_can_buy_a_session.feature

```diff
@@ -7,16 +7,17 @@ Feature: User can buy a subscription
 
 	Background:
 		Given the following user exists
-			| first_name | last_name | email         | password |
-			| John       | Doe       | john@mail.com | password |
+			| first_name | last_name | email         | password | role    |
+			| John       | Doe       | john@mail.com | password | trainee |
 
 		And the following sessions exist
-			| title     | start_date          | end_date            | status    |
-			| Crossfit  | 2019-02-01 15:00:00 | 2019-02-01 15:30:00 | available |
+			| title    | start_date          | end_date            | status    |
+			| Crossfit | 2019-02-01 15:00:00 | 2019-02-01 15:30:00 | available |
 		And the time is 2019 1 30 10:00:00
 
 	Scenario: Trainee can successfully buy a subscription
 		And I am logged in as 'john@mail.com'
+		Then show me the page
```

Comment:
Why do we have this here?
Remove it if it has no purpose

Reply:
Done